### PR TITLE
feat: Bump gitea to 6.0.1

### PR DESCRIPTION
**Pull request by mesosphere-teamcity**
This upgrades gitea to 6.0.1 to use the latest upstream chart gitea 6.0.1.

### CODE REVIEW

**Comment by gracedo** [comment]
@mesosphere-mergebot test all

**Comment by gracedo** [comment]
@mesosphere-mergebot test all

**Review by gracedo — APPROVED** [review]

File: services/gitea/6.0.1/defaults/kustomization.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
----
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- cm.yaml
+  - cm.yaml
```
Comment (mikolajb) [inline services/gitea/6.0.1/defaults/kustomization.yaml:4]: Why? 🙂

**Review by mikolajb — APPROVED** [review]

File: services/gitea/6.0.1/defaults/kustomization.yaml (hunk 1 above)
Comment (gracedo) [reply services/gitea/6.0.1/defaults/kustomization.yaml:4]: I think the automation is doing some reformatting but it'll probably just happen this first bump 😅

**Pull request merged**